modules/vault-elb/main.tf

```diff
@@ -24,6 +24,18 @@ resource "aws_elb" "vault" {
   security_groups = [aws_security_group.vault.id]
   subnets         = var.subnet_ids
 
+  # optional access_logs creation  
+  dynamic "access_logs" {
+    for_each = var.access_logs == null ? [] : ["once"]
+
+    content {
+      enabled       = lookup(access_logs.value, "enabled", lookup(access_logs.value, "bucket", null) != null)
+      bucket        = lookup(access_logs.value, "bucket", null)
+      bucket_prefix = lookup(access_logs.value, "bucket_prefix", null)
+      interval      = lookup(access_logs.value, "interval", 60)
```

Comment:
NIT: should `interval` also default to `null`?

Reply:
I don't think interval should be null, when we can assign a sane value, which in this case we can, we shouldn't require the user to explicitly state what the interval is.

Reply:
Same thing here: [`interval`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elb#interval) is listed as an optional. If the user doesn't provide it, and we set it to `null`, we automatically get Terraform's default, rather than having to provide our own.

Reply:
corrected